AirCasting/Notes/AddNote/AddNoteView.swift

```diff
@@ -32,7 +33,12 @@ private extension AddNoteView {
     }
     
     var noteField: some View {
-        createNoteTextField(binding: $viewModel.noteText)
```

Comment:
What was wrong with this function?

Reply:
I will back to the previous implementation. I thought that will be better to create a TextView directly here.